README.md

```diff
@@ -31,6 +31,24 @@ Optional. A folder to write the output files to. Will create if it does not exis
 `-s` or `--subfolder`
 Optional. If used, generates a folder for each tiff being split and puts the individual tiffs in the subfolder instead of alongside the parent tiff.
 
+### split-tiled
+
+`$ java -jar image-utils-0.1.0-standalone.jar split-tiled [args]`
+
+Splits a tiled tiff into individual tiffs.
+Unlike split-hyperstack which ouputs [originalfilename]_[channel].tiff, 
+split tiled outputs the files as [channel].tiff, so it is recommended 
+to use `--subfolder`. Accepts the following options:
```

Comment:
What happens if split-tiled is run without -s on a folder with multiple tiled tiffs that each have the same channels?

Separately, I think it would be good to be consistent with behavior across commands. I'd be fine with making -s or --subfolder and only channel names for output tiffs be the normal behavior and instead have an option --flat or something similar that outputs with the original filename and does everything in one folder.

Reply:
Good point. I went with not modifying -s FALSE behavior, and made it consistent using tiled tiffs as well. And then changed -s TRUE behavior so that the files in subfolders are not named with the base file name, and only are channelname.tiff

Thoughts?

Reply:
That sounds good to me! I think it's fine if the files in subfolders don't also have the basename. Thanks dude!